payments/swish/redirect.md

```diff
@@ -18,21 +18,19 @@ sidebar:
 
 {% include alert-development-section.md %}
 
->Swish is an one-phase payment instrument supported by the major Swedish banks.
- In the redirect e-commerce scenario, Swedbank Pay performs a payment that the
- payer confirms using her Swish mobile app.
- The consumer initiates the payment by supplying the Swish registered mobile
- number (MSISDN), connected to the Swish app.
+{% include jumbotron.html body="**Swish Payments Redirect** is where Swedbank
```

Comment:
Please keep the sentence "Swish is an one-phase payment instrument supported by the major Swedish banks". It should also be made apparent how Redirect differs from Seamless View.